classes/class-p4-user.php

```diff
@@ -73,7 +73,7 @@ public function name() : string {
 			if ( $this->is_fake ) {
 				return (string) $this->display_name;
 			} else {
-				return parent::name();
+				return parent::name() ?? '';
```

Comment:
You could also solve this by making the return type nullable (`name(): ?string`), which is maybe more straightforward as then the signature is the same as `name()` in the parent.